src/dialogs/config-flow/styles.ts

```diff
@@ -22,6 +22,9 @@ export const configFlowContentStyles = css`
     text-transform: var(--mdc-typography-headline6-text-transform, inherit);
     box-sizing: border-box;
   }
+  h2.end-space {
+    padding-inline-end: 72px;
+  }
```

Comment:
Can we use css variable to set this instead of a new property?
```
h2 {
   padding-inline-end: var(--title-padding-inline-end, 0);
}
```

And `--title-padding-inline-end` can be set in `dialog-data-entry-flow.ts`. What do you think?

Reply:
Yes sure thats also an option.
I am not sure if it's worth the effort to change it again :slightly_smiling_face: 

In my opinion we should refactor the whole dialog-data-entry-flow. Each step does have a header, why not use `ha-dialog-header` for example, that would solve that problem and would set the close botton on the left where it should be. 

But I don't want to do such a change now as a beta patch

Reply:
I agree, let's merge it to fix the issue during beta and improve the flows later